lib/engine/game/g_steam_over_holland/entities.rb

```diff
@@ -0,0 +1,189 @@
+                count: 1,
+                free: true,
+                tiles: [],
+                hexes: [],
+                closed_when_used_up: true,
+              },
+            ],
+          },
+          {
+            name: '2. Koninklijk Korps van Ingenieurs',
+            sym: 'KKI',
+            value: 40,
+            revenue: 10,
+            desc: 'Pays the costs for crossing a river one time. Closes after use.',
+            abilities: [
+              {
+                type: 'tile_discount',
+                owner_type: 'corporation',
+                discount: 50,
+                terrain: 'water',
+                count: 1,
+                hexes: [],
+              },
+            ],
+          },
+          {
+            name: '3. Veerdienst Enkhuizen - Stavoren',
+            sym: 'VES',
+            value: 75,
+            revenue: 15,
+            desc: 'Fl. 20 bonus revenue for the owning company if it uses the ferry across the Zuiderzee at Enkhuizen (D9).',
+            abilities: [
+              {
+                type: 'hex_bonus',
+                owner_type: 'corporation',
+                hexes: ['D9'],
+                amount: 20,
+              },
+            ],
+          },
+          {
+            name: '4. Veerdienst Vlissingen - Londen',
+            sym: 'VVL',
+            value: 75,
+            revenue: 15,
+            desc: 'Fl. 20 bonus for the owning company each time it starts or ends a route in Vlissingen (J1).',
+            abilities: [
+              {
+                type: 'hex_bonus',
+                owner_type: 'corporation',
+                hexes: ['J1'],
+                amount: 20,
+              },
+            ],
+          },
+          {
+            name: '5. Rijks Waterstaat',
+            sym: 'RW',
+            value: 80,
+            revenue: 20,
+            desc: 'Allows the owning company to lay one station token for free. Closes after use.',
+            abilities: [
+              {
+                type: 'token',
+                price: 0,
+                from_owner: true,
+                special_only: true,
+                connected: true,
+                hexes: [],
+              },
+            ],
+          },
+          {
+            name: '6. Werkspoor',
+            sym: 'W',
+            value: 100,
+            revenue: 20,
+            desc: 'Gives a 10% discount on all trains the owning company buys from the bank.',
+            abilities: [
+              {
+                type: 'train_discount',
+                discount: 0.1,
+                trains: %w[3 4 5],
+              },
+            ],
+          },
+          {
+            name: '7. Koninklijke Ondersteuning ',
+            sym: 'KO',
+            value: 100,
+            revenue: 10,
+            desc: 'Comes with a share of the NRS',
+            abilities: [{ type: 'shares', shares: 'NRS_1' }],
+          },
+        ].freeze
+
+        def game_corporations
```

Comment:
I did this because I'm going to need to make the float_percentage for one of the corporations variable depending on which privates are in the game.

I think this is the first incremental cap game implemented that has a share given to a player in the initial auction. To my mind, it looks weird for a corp to say "10% to float" when it will still require the purchase of a president's certificate, so I wanted to be able to change the float_percentage to 30 if that private is in the game. I just haven't implemented all that code yet.